proto/types/types.proto

```diff
@@ -169,3 +170,17 @@ message SignedHeader {
   Header header = 1;
   Commit commit = 2;
 }
+
+message ValidatorSet {
+  repeated Validator validators         = 1;
+  Validator          proposer           = 2;
+  int64              total_voting_power = 3;  // TODO: change to uint64
```

Comment:
why not change it to uint64 now? we should make this field private eventually as that's the current behavior.

Reply:
Yes I can make that change 

For making it private, I don't believe I can as I would not be able to set it from the types pkg. Correct me if wrong

Reply:
you would need to copy the method that you have on tmtypes that populates the total voting power